components/board/fake/board.go

```diff
@@ -226,6 +226,11 @@ func (b *Board) WriteAnalog(ctx context.Context, pin string, value int32, extra
 	return nil
 }
 
+// StreamTicks starts a stream of digital interrupt ticks.
+func (b *Board) StreamTicks(ctx context.Context, interrupts []string, ch chan board.Tick, extra map[string]interface{}) error {
```

Comment:
note to self - we will need to implement this to get the gpio motor tests to pass with the new ticks stream implementations, .. or front load refactoring those tests TODO:Rand so that they use injected boards from within component/motor/gpio/*_test.go so we don't have a bad time in the future.